### DISCUSSION TITLE
Showing some details of YoloV5.

### DISCUSSION THREAD

**Discussion by yyccR** [ultralytics/yolov5#6717] [Show and tell]
# yolov5 release6.0 struture:
![image](https://user-images.githubusercontent.com/14156500/154876030-a49b19c9-abf3-430c-b814-a321377e7885.png)

# box predict compare with yolov3:
![image](https://user-images.githubusercontent.com/14156500/154876204-23532257-d301-4efc-b67f-6222d34f38fb.png)

![image](https://user-images.githubusercontent.com/14156500/154876226-e5ad96e8-aa16-414c-b3bc-7bd9a1af546a.png)

![image](https://user-images.githubusercontent.com/14156500/154876246-81dc4059-f58d-4964-9675-a54933131944.png)

# positive sampling
![image](https://user-images.githubusercontent.com/14156500/154876318-ba889843-6cc1-4769-b2c7-4620c0150ce6.png)

# box loss
![image](https://user-images.githubusercontent.com/14156500/154876427-6b8f5965-2787-43c0-adf1-0027faf1838f.png)

![image](https://user-images.githubusercontent.com/14156500/154876471-f4563b0d-5a4f-460a-b65c-e346f491ed6d.png)

![image](https://user-images.githubusercontent.com/14156500/154876498-196932ef-a627-4820-8a86-cf76c38210f4.png)

# objectness loss

![image](https://user-images.githubusercontent.com/14156500/154876512-843e57de-e6d6-4f53-8f4f-d08f78595922.png)

# class loss:

![image](https://user-images.githubusercontent.com/14156500/154876527-f8a8881f-e2f2-4151-8b8c-50b39e717e8f.png)

# more details can be found on this tutorial pdf: 

```https://github.com/yyccR/yolov5_in_tf2_keras/blob/master/yolov5_bilibili_talk_ppt.pdf```

**Comment by glenn-jocher** [comment]
@yyccR super summary. Thank you for your contributions!
